lib/vmdb/initializer.rb

```diff
@@ -20,7 +20,7 @@ def self.init
 
       # Rails console needs session store configured
       if defined?(Rails::Console)
-        MiqUiWorker.preload_for_worker_role
+        MiqUiWorker.configure_secret_token(SecureRandom.hex(64))
```

Comment:
`preload_for_console`, so it's more obvious they need to be kept in sync?

Well, not in sync -- that's the point of the change -- but that if you're changing one, you should consider whether your change also applies to the other.